importer/generateFont.js

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// @ts-check
+
+const fs = require("fs/promises");
+const mkdirp = require('mkdirp');
+const path = require("path");
+const { promisify } = require('util');
+const glob = promisify(require('glob'));
+const process = require("process");
+const argv = require("yargs").boolean("selector").default("selector", false).argv;
+const _ = require("lodash");
+const fantasticon = require('fantasticon');
+
+const SRC_PATH = argv.source;
+const DEST_PATH = argv.dest;
+const ICON_TYPE = argv.iconType;
+const CODEPOINTS_FILE = argv.codepoints;
+
+if (!SRC_PATH) {
+    throw new Error("SVG source folder not specified by --source");
+}
+if (!DEST_PATH) {
+    throw new Error("Output destination folder not specified by --dest");
+}
+if (!(ICON_TYPE === 'Filled' || ICON_TYPE === 'Regular' || ICON_TYPE === 'OneSize')) {
```

Comment:
Rather than OneSize we should probably keep with the previous naming pattern.... I believe it was Resizable?